tldr.py

```diff
@@ -362,4 +362,7 @@ def main():
 
 
 if __name__ == "__main__":
-    main()
+    try:
+        main()
+    except KeyboardInterrupt:
+        print('\nAbort!')
```

Comment:
"User Exited"  might be better than abort. Also, it might be better to use ```sys.exit``` here.